app/models/user.rb

```diff
@@ -442,28 +442,39 @@ def self.diaspora_id_host
   end
 
   def seed_aspects
-    self.aspects.create(:name => I18n.t('aspects.seed.family'))
-    self.aspects.create(:name => I18n.t('aspects.seed.friends'))
-    self.aspects.create(:name => I18n.t('aspects.seed.work'))
-    aq = self.aspects.create(:name => I18n.t('aspects.seed.acquaintances'))
-
-    if AppConfig.settings.autofollow_on_join?
-      begin
-        default_account = Person.find_or_fetch_by_identifier(AppConfig.settings.autofollow_on_join_user)
-        share_with(default_account, aq)
-      rescue DiasporaFederation::Discovery::DiscoveryError
-        logger.warn "Error auto-sharing with #{AppConfig.settings.autofollow_on_join_user}
+    aspects.create(name: I18n.t("aspects.seed.family"))
+    aspects.create(name: I18n.t("aspects.seed.friends"))
+    aspects.create(name: I18n.t("aspects.seed.work"))
+    acquaintances = aspects.create(name: I18n.t("aspects.seed.acquaintances"))
+
+    acquaintances.tap do |aq|
+      if AppConfig.settings.autofollow_on_join?
+        autofollow_user = AppConfig.settings.autofollow_on_join_user
+        autofollow_accounts = Array.wrap(AppConfig.settings.autofollow_on_join_accounts)
+        autofollow_accounts.push(autofollow_user) if autofollow_user.present?
+
+        begin
+          autofollow_accounts.uniq.each { |user_id| follow_account(user_id, aq) } if autofollow_accounts.present?
```

Comment:
```suggestion
          autofollow_accounts.uniq.each { |user_id| follow_account(user_id, aq) }
```

`Array.wrap` will ensure it's an array already, the `#push` call above would already fail if it's not and  `each` on an empty array is a noop. So I think we can do without the `if` condition :)

Reply:
Resolved here: [25ae7a9](https://github.com/diaspora/diaspora/pull/8430/commits/25ae7a9819945b755e76ed6122b650531a8ec957).